src/util/CrashHandlerUnix.h

```diff
@@ -70,11 +72,11 @@ static void crashHandler(int sig) {
     time_t curtime = time(0);
     char stime[128];
     strftime(stime, sizeof(stime), "%Y%m%d-%H%M%S", localtime(&curtime));
-    Path errorlogPath = Util::getCacheSubfolder(ERRORLOG_DIR);
+    auto errorlogPath = Util::getCacheSubfolder(ERRORLOG_DIR);
     errorlogPath /= std::string("errorlog.") + stime + ".log";
-    ofstream fp(errorlogPath.str());
+    ofstream fp(errorlogPath);
     if (fp) {
-        g_warning("[Crash Handler] Wrote crash log to: %s", errorlogPath.str().c_str());
+        g_warning("[Crash Handler] Wrote crash log to: %s", errorlogPath.c_str());
```

Comment:
The two implementations result in the same thing on *nix. We can safely assume that this is on a *nix system due to the filename.

Reply:
I'm interested, what happens on windows systems, because it is a wchar then. But if g_warning can handle this it is safer than converting to a local locale string.

Reply:
Windows uses `CrashHandlerWindows`. The whole file `CrashHandlerUnix` isn't included on Windows.